### ISSUE TITLE
bug(aria): aria-invalid is missing for email and required combination

### ISSUE THREAD

**Issue by richarddesan** [angular/components#32739]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The aria-invalid attribute is missing for matInput when an email and a required validation are set

### Reproduction

StackBlitz link: 
https://stackblitz.com/edit/gqo2czqp?file=src%2Fexample%2Finput-errors-example.html
Steps to reproduce:
1. focus and blur the field or
2. submit

### Expected Behavior

An aria-invalid="true" attribute should be on the input field

### Actual Behavior

Missing aria-invalid attribute

### Environment

Angular CLI       : 21.0.4
Angular           : 21.0.6
Node.js           : 22.17.1
Package Manager   : pnpm 10.27.0
Operating System  : win32 x64

**Comment by crisbeto** [comment]
Looks like it was intentional in order to make the component less noisy when using a screen reader. See #21609.
